.github/workflows/ruby.yml

```diff
@@ -30,7 +30,8 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby-version }}
-          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+          bundler-cache: true
+          bundler-args: --without development
```

Comment:
This doesn't seem to be working unless we somehow end up installing all the quickdraw gems from the its Gemfile here. 🤔